### DISCUSSION TITLE
Latest E3V2-UBL-BLTouch-10x10-HS-v4.2.2, issue with mesh... (BLTouch 3.1)

### DISCUSSION THREAD

**Discussion by VashBaldeus** [Jyers/Marlin#559] [Q&A] [answered]
[Screenshot](https://i.imgur.com/NZgerOD.jpg), as you can see, it does 65/100 points and stops there.
[Result of try Mesh(image)](https://i.imgur.com/eOHfaqo.jpg), is this normal that 0.01 shows black.. while other parts with 0.01 show very dark green..

**Comment by smooreace** [comment, marked as answer]
As far as colors go, yes.. That is correct. Green is the bed being high and
red is the bed being low. And there are MANY discussions ongoing about mesh
points now being "complete". Search is your friend.
